platform/uboot/uboot.mk

```diff
@@ -102,6 +133,50 @@ $(PRODUCT_OUT)/bootloader-sd.img: $(UBOOT_BINARY)
 	dd if=/dev/null of=$@ bs=1 count=1 seek=$$(( 2048 * 1024 - 256 * 512 ))
 endif
 
+ifeq ($(PRODUCT_BOARD_PLATFORM),rockchip)
+$(UBOOT_BINARY_SD_EMMC): $(UBOOT_BINARY)
+	mkdir -p $(UBOOT_OUT)
+	$(UMAKE) $(UBOOT_DEFCONFIG)
+	PATH=/usr/bin:/bin $(UBOOT_SRC)/scripts/kconfig/merge_config.sh -m -O $(UBOOT_OUT)/ $(UBOOT_OUT)/.config $(UBOOT_FRAGMENTS) $(UBOOT_FRAGMENT_EMMC)
+	$(UMAKE) olddefconfig
+	$(UMAKE) KCFLAGS="$(UBOOT_KCFLAGS)"
+	cp $(UBOOT_BINARY) $@
+
+$(UBOOT_IDBLOADER): $(UBOOT_BINARY)
+	$(ROCKCHIP_FIRMWARE_DIR)/tools/mkimage -n rk3399 -T rksd -d $(ROCKCHIP_FIRMWARE_DIR)/bin/$(DDR_BLOB) $@
+	cat $(ROCKCHIP_FIRMWARE_DIR)/bin/$(MINILOADER_BLOB) >> $@
+
+$(UBOOT_TRUST): $(UBOOT_BINARY)
+	mkdir -p $(UBOOT_OUT)/bin
+	(cp -r $(RK_BIN_DIR) $(UBOOT_OUT)/bin && \
+	cp $(RKTRUST_DIR)/$(RKTRUST_INI) $(UBOOT_OUT) && \
+	cp $(ROCKCHIP_FIRMWARE_DIR)/$(TRUST_MERGER) $(UBOOT_OUT)/$(TRUST_MERGER) && \
+	cd $(UBOOT_OUT) && \
+	$(TRUST_MERGER) $(RKTRUST_INI))
+
+$(PRODUCT_OUT)/bootloader-sd.img: $(UBOOT_IDBLOADER) $(UBOOT_TRUST)
```

Comment:
Can you combine next 3 rules into single?
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_node/make_38.html